From 8fcb9b0c0481e965ab1dda1a1ac8406f388c5181 Mon Sep 17 00:00:00 2001
From: Cherry Chen <cherry.chen@rock-chips.com>
Date: Thu, 23 Aug 2018 17:40:28 +0800
Subject: [PATCH] arm64: dts: rockchip: Put the order of aloop card backwards
 for rk3308 board

For keep other sounds card order, the aloop card index set 7.

Change-Id: I7336ac17dfeba67fc0c5c52429d9a74e9856e7e9
Signed-off-by: Cherry Chen <cherry.chen@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308-ai-va-v10.dts | 2 +-
 arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi  | 2 +-
 arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi  | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308-ai-va-v10.dts b/arch/arm64/boot/dts/rockchip/rk3308-ai-va-v10.dts
index f416d59b0ccd..dd519ce7763d 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-ai-va-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3308-ai-va-v10.dts
@@ -13,7 +13,7 @@
 	compatible = "rockchip,rk3308-ai-va-v10", "rockchip,rk3308";
 
 	chosen {
-		bootargs = "earlycon=uart8250,mmio32,0xff0c0000 swiotlb=1 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=squashfs rootwait";
+		bootargs = "earlycon=uart8250,mmio32,0xff0c0000 swiotlb=1 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=squashfs rootwait snd_aloop.index=7";
 	};
 
 	adc-keys0 {
diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
index d10f5442550b..40f44013fe5b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
@@ -12,7 +12,7 @@
 	compatible = "rockchip,rk3308-evb", "rockchip,rk3308";
 
 	chosen {
-		bootargs = "earlycon=uart8250,mmio32,0xff0c0000 swiotlb=1 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=squashfs rootwait";
+		bootargs = "earlycon=uart8250,mmio32,0xff0c0000 swiotlb=1 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=squashfs rootwait snd_aloop.index=7";
 	};
 
 	adc-keys0 {
diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi b/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi
index 8028d5f094ce..4e8885d73e33 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi
@@ -11,7 +11,7 @@
 	compatible = "rockchip,rk3308-evb-v11", "rockchip,rk3308";
 
 	chosen {
-		bootargs = "earlycon=uart8250,mmio32,0xff0c0000 swiotlb=1 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=squashfs rootwait";
+		bootargs = "earlycon=uart8250,mmio32,0xff0c0000 swiotlb=1 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=squashfs rootwait snd_aloop.index=7";
 	};
 
 	adc-keys {
-- 
2.35.3

